<launch>


  <node name="pick_place_server" pkg="moveit_interface" type="pick_place_server"
    respawn="false" output="screen"/>


</launch>
